src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -15,6 +15,17 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
+  <!-- After building, copy SDK archive to assets root. -->
+  <Target Name="CopySdkArchive"
+          AfterTargets="Build">
+    <ItemGroup>
+      <_SdkTarball Include="$(ArtifactsAssetsDir)**/dotnet-sdk-*$(ArchiveExtension)" />
```

Comment:
We already have the `DetermineSourceBuiltSdkVersion` that finds the tarball. Depend on that target instead of globbing for the tarball yet again.

Reply:
Yeah, this was one of the lower-pri items on my list and didn't get to it. Will fix.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86